create-kafka-queues/create-topics.sh

```diff
@@ -54,9 +54,17 @@ else
 
   #Search for all environmental variables starting with the word: TOPIC
   for topic in "${!TOPIC@}"; do
```

Comment:
I get what the `!` does, but what does the `@` do? From googling, all I get is arrays.

Reply:
This is searching for all env variables that have "TOPIC" at the start. This will create an array of these values that we then use in the rest of the script. These values are created through the `create-kafka-queues` values.yaml file.